GSServer.py

```diff
@@ -71,12 +71,9 @@ def start_server(args, m=MVelKeepConfig()):
         log.error("Failed to load scenario")
         return
 
-    sync_global = TickSync(rate=sim_config.traffic_rate, realtime=True, block=True, verbose=False, label="EX")
+    sync_global = TickSync(rate=sim_config.traffic_rate, realtime=True, block=True, verbose=False, label="EX", usr_input=sim_config.wait_for_input)
     sync_global.set_timeout(sim_config.timeout)
 
-    if sim_config.wait_for_input:
```

Comment:
by removing this you break the wait for input when `--no-dash` is used.

I thought of keeping this and depending on `--no-dash`, ask for terminal enter or show window and ask for enter there.

Reply:
works now.